docs/hardware.rst

```diff
@@ -427,28 +449,19 @@ Intel 7th-gen NUC
 We have tested and can recommend the `NUC7i5BNH <https://ark.intel.com/content/www/us/en/ark/products/95067/intel-nuc-kit-nuc7i5bnh.html>`__.
 
 The NUC7i5BNH has soldered-on wireless components, which cannot easily be
-removed. For security reasons, we recommend that you take the following steps
-to disable wireless functionality:
-
-- before installation of the RAM and storage, disconnect the wireless antennae
-  leads.
+removed. Before installation of the RAM and storage, we recommend that you
+disconnect the wireless antennae leads:
 
 |NUC7 leads|
 
-- before the initial OS installation, boot into the BIOS by pressing **F2** at
-  startup, navigate to **Advanced > Devices > Onboard Devices**, and disable
-  unwanted hardware - everything except **LAN**.
-
-|Visual Bios|
+Before the initial OS installation, boot into the BIOS by pressing **F2** at
+startup and adjust the system configuration:
 
+- Under **Advanced ▸ Devices ▸ Onboard Devices**, disable all onboard devices
+  other than LAN: audio, audio DSP, microphone, Thunderbolt, WLAN, Bluetooth,
+  and SD card.
```

Comment:
I don't have a NUC7 at hand, but I noticed that these BIOS instructions don't include the recommendations we specify for the other NUC models:

> - Under **Advanced ▸ Security**, disable SGX support, which is not used by  SecureDrop and may be targeted by active CPU exploits.
> - Under **Advanced ▸ Boot ▸ Secure Boot**, uncheck the **Secure Boot** checkbox.

If these recommendations are applicable on NUC7 as well, and someone can validate that these are the correct menus, I can add them here as well.